src/pyOpenMS/pyopenms/_dataframes.py

```
@@ -75,6 +99,42 @@ class _ConsensusMapDF(_ConsensusMap):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
+    def get_df_columns(self, columns: str = 'default') -> List[str]:
```

Ruff (0.14.7)

Unused method argument: `columns`

(ARG002)